### ISSUE TITLE
NullPointerException in JFileChooser initialization on Linux

### ISSUE THREAD

**Issue by FSchumacher** [weisJ/darklaf#160]
On linux (ubuntu 20.04 running OpenJDK 11.0.7) running JMeter I get a NullPointerException at
https://github.com/weisJ/darklaf/blob/0716769bfa954d153cc8d1b53a6ba06f29b38544/core/src/main/java/com/github/weisj/darklaf/components/tooltip/TooltipAwareButton.java#L69 

To reproduce start JMeter and use File->Templates... to open the Templates dialog. Select `Recording with Think Time` and press `Create`. In the log file `jmeter.log` you should see a stack trace containing:

```
Caused by: java.lang.NullPointerException
>-------at com.github.weisj.darklaf.components.tooltip.TooltipAwareButton.getToolTipContext(TooltipAwareButton.java:69) ~[darklaf-core-2.1.0.jar:2.1.0]
>-------at com.github.weisj.darklaf.components.tooltip.TooltipAwareButton.updateUI(TooltipAwareButton.java:62) ~[darklaf-core-2.1.0.jar:2.1.0]
>-------at javax.swing.AbstractButton.init(AbstractButton.java:2136) ~[?:?]
>-------at javax.swing.JButton.<init>(JButton.java:131) ~[?:?]
>-------at javax.swing.JButton.<init>(JButton.java:85) ~[?:?]
>-------at javax.swing.JButton.<init>(JButton.java:116) ~[?:?]
>-------at com.github.weisj.darklaf.components.tooltip.TooltipAwareButton.<init>(TooltipAwareButton.java:57) ~[darklaf-core-2.1.0.jar:2.1.0]
>-------at com.github.weisj.darklaf.ui.filechooser.DarkFileChooserUI.createUpFolderButton(DarkFileChooserUI.java:317) ~[darklaf-core-2.1.0.jar:2.1.0]
>-------at com.github.weisj.darklaf.ui.filechooser.DarkFileChooserUI.createTopButtonArea(DarkFileChooserUI.java:262) ~[darklaf-core-2.1.0.jar:2.1.0]
>-------at com.github.weisj.darklaf.ui.filechooser.DarkFileChooserUI.createTopPanel(DarkFileChooserUI.java:236) ~[darklaf-core-2.1.0.jar:2.1.0]
>-------at com.github.weisj.darklaf.ui.filechooser.DarkFileChooserUI.installComponents(DarkFileChooserUI.java:90) ~[darklaf-core-2.1.0.jar:2.1.0]
>-------at javax.swing.plaf.basic.BasicFileChooserUI.installUI(BasicFileChooserUI.java:209) ~[?:?]
>-------at com.github.weisj.darklaf.ui.filechooser.DarkFileChooserUIBridge.installUI(DarkFileChooserUIBridge.java:137) ~[darklaf-core-2.1.0.jar:2.1.0]
>-------at javax.swing.JComponent.setUI(JComponent.java:685) ~[?:?]
>-------at javax.swing.JFileChooser.updateUI(JFileChooser.java:1825) ~[?:?]
>-------at javax.swing.JFileChooser.setup(JFileChooser.java:382) ~[?:?]
>-------at javax.swing.JFileChooser.<init>(JFileChooser.java:348) ~[?:?]
>-------at javax.swing.JFileChooser.<init>(JFileChooser.java:295) ~[?:?]
>-------at org.apache.jmeter.gui.util.FileDialoger.<clinit>(FileDialoger.java:42) ~[ApacheJMeter_core.jar:5.3-SNAPSHOT]
>-------... 63 more
```

The corresponding lines in JMeter `FileDialoger` are

https://github.com/apache/jmeter/blob/b6d11d79d905d0c099732bb928d2372fd1388981/src/core/src/main/java/org/apache/jmeter/gui/util/FileDialoger.java#L42

so I am not sure, that we can do anything about it on our side.

**Comment by weisJ** [comment]
Duplicate of #157. It's already fixed in `2.1.1`

**Comment by FSchumacher** [comment]
> Duplicate of #157. It's already fixed in `2.1.1`

Wow, you are fast. Thanks. (Should have pulled before reporting.)
